Tests/CatalogItems/Get-RsDataSource.Tests.ps1

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+Function Get-ExistingDataExtension
+{
+        $proxy = New-RsWebServiceProxy
```

Comment:
nit: double check the indentation seems to be off  (seems to be double indentation)